frontend/src/styles/globals.css

```
@@ -24,3 +24,13 @@
     @apply bg-background text-foreground;
   }
 }
+/* width */
+::-webkit-scrollbar {
+  width: 8px;
+}
+ 
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #d500f9; 
```

```suggestion
  background: #D926D9; 
```
should use secondary color